### DISCUSSION TITLE
Want to Spilt One page to several pages based on the specific Lines

### DISCUSSION THREAD

**Discussion by Nathan1992212** [jsvine/pdfplumber#547] [Ask for help with specific PDFs]
Hi,
[China_ptent.pdf](https://github.com/jsvine/pdfplumber/files/7603511/China_ptent.pdf)

This is the Pdf I want to share. I want to extract the information according to the explanation of the  image.
![f8ad6b0c9012849edf602254a928093](https://user-images.githubusercontent.com/94977430/143454894-2866e008-b715-482c-8289-781cabec2559.png).
Due to this, could you provide me with some solution. Is that possible way that I can extract the data based on the appearance of the black straight lines on each pdf's page.

**Comment by jsvine** [comment]
Hi @Nathan1992212, and thanks for your interest in this library. You should indeed be able to extract data based on the lines you see in the PDF. The specific solution will depend on the particulars of the PDF, but the general approach will likely be:

- Use `my_page_var.lines` to see the information about all lines on the page. You may also want to check `.curves` for line-like objects that don't quite conform to `pdfminer.six`'s line-detection rules.
- Identify the lines that match your criteria.
- Use information those lines in conjunction with `.crop((x0, top, x1, bottom))` to select each region of the PDF you care about.

**Reply by Nathan1992212** [reply]
thank you for your help.I will try to extact it based on your suggestion
